examples/playground/src/pages/components/visualization/Table/components/MutatorTable.js

```diff
@@ -0,0 +1,223 @@
+// External Requirements
+var React                  = require('react');
+var MortarJS               = require('../../../../../app-container').MortarJS;
+var moment                 = require('moment');
+
+// Bricks
+var Br                     = MortarJS.require('components', 'Row', 'Column', 'Table');
+
+// Stores
+var FormStore              = MortarJS.Stores.FormStore;
+
+// Mixins
+var ResourceComponentMixin = MortarJS.Mixins.ResourceComponentMixin;
+
+/**
+ * Mutator Table
+ *
+ * @type {*|Function}
+ */
+var MutatorTable = React.createClass({
+	mixins: [ResourceComponentMixin],
+
+	getInitialState: function() {
+		return {
+			params          : {},
+			formIsValid     : true,
+			workingResource : [
+				{
+					'id'         : '0',
+					'first_name' : 'Darth',
+					'last_name'  : 'Vader',
+					'username'   : 'lilorphanannie',
+					'hands'      : 0,
+					'occupation' : 'Sith Lord',
+					'email'      : 'lilorphanannie@galaxyfarfaraway.com',
+					'birthday'   : '1931-01-17T18:35:24+00:00',
+					'vehicles'   : [
+						[
+							'TIE Fighter'
+						]
+					]
+				},
+				{
+					'id'         : '1',
+					'first_name' : 'Leia',
+					'last_name'  : 'Organa',
+					'username'   : 'lorgana',
+					'hands'      : 2,
+					'occupation' : 'Rebel',
+					'email'      : 'lorgana@galaxyfarfaraway.com',
+					'birthday'   : '1956-10-21T18:35:24+00:00',
+					'vehicles'   : [
+						[
+							'Alderaan Cruiser'
+						],
+						[
+							'Tantive IV'
+						]
+					]
+				},
+				{
+					'id'         : '2',
+					'first_name' : 'Luke',
+					'last_name'  : 'Skywalker',
+					'username'   : 'lskywalker',
+					'hands'      : 1,
+					'occupation' : 'Jedi',
+					'email'      : 'lskywalker@galaxyfarfaraway.com',
+					'birthday'   : '1951-09-25T18:35:24+00:00',
+					'vehicles'   : [
```

Comment:
@leayousoufov Why are vehicles an array of single-index arrays and not an array of strings?